packages/sit-onyx/src/components/OnyxButton/types.ts

```diff
@@ -0,0 +1,31 @@
+export type ButtonProps = {
+  /**
+   * The text content of the button.
+   */
+  label: string;
+  /**
+   * If the button should be disabled or not.
+   */
+  disabled?: boolean;
+  /**
+   * The button type.
+   */
+  type?: ButtonType;
+  /**
+   * The color of the button.
+   */
+  color?: ButtonColor;
+  /**
+   * The style of the button.
+   */
+  variant?: ButtonVariant;
```

Comment:
```suggestion
  mode?: ButtonMode;
```
I talked to Jannick and we decided to use "mode" instead, as "type" (which is used in figma) is already reserved